backend/jest.config.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  testEnvironment: 'node',
+  transform: {
+    '^.+\\.tsx?$': 'ts-jest',
+  },
+  moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'json', 'node'],
+  testRegex: '(/__tests__/.*|(\\.|/)(test|spec))\\.(ts|js)x?$',
+  coverageDirectory: 'coverage',
+  collectCoverageFrom: ['src/**/*.{ts,tsx,js,jsx}', '!src/**/*.d.ts'],
+  moduleNameMapper: {
+    '^@controllers(.*)$': '<rootDir>/src/controllers$1',
```

Comment:
Maybe is there an option to create regexp which takes all folders after <root> and creates these paths automatically. 

Did you add this configuration because tsconfig aliases were not visible when running the tests?

Reply:
it's not necessary any longer, removed whole moduleNameMapper property